### ISSUE TITLE
[Airbrake] [Mhlab] Cannot read property '0' of undefined

### ISSUE THREAD

**Issue by hotzenklotz** [scalableminds/webknossos#2547]
**Airbrake error:** [#7518](https://scm.airbrake.io/projects/95438/groups/2205863759280137518)
**Airbrake project:** webknossos

**Error type:** `TypeError`
**Error message:** `` Cannot read property '0' of undefined ``
**Where:** `<no information>`
**Occurred at:** May 02, 2018 12:23:40 UTC
**First seen at:** May 02, 2018 12:17:31 UTC
**Occurrences:** 2 ( since last deploy on `<no information>`)
**Severity:** `error`

**URL:** [https://webknossos.brain.mpg.de/annotations/Explorational/5ae9ac2528000055006406b2#19628,17226,0,3,20.78](https://webknossos.brain.mpg.de/annotations/Explorational/5ae9ac2528000055006406b2#19628,17226,0,3,20.78)
**File:** `/PROJECT_ROOT/assets/bundle/main.js?nocache=68a5d6619b5009b18f99dd111dc7c2dd32f0fae5`

**Backtrace:**
```
/PROJECT_ROOT/assets/bundle/main.js?nocache=68a5d6619b5009b18f99dd111dc7c2dd32f0fae5:1:in t.b.calculateUnzoomedAnchorPoint
/PROJECT_ROOT/assets/bundle/main.js?nocache=68a5d6619b5009b18f99dd111dc7c2dd32f0fae5:1:in t.b.updateDataTextures
/PROJECT_ROOT/assets/bundle/main.js?nocache=68a5d6619b5009b18f99dd111dc7c2dd32f0fae5:1:in Object.update
/PROJECT_ROOT/assets/bundle/main.js?nocache=68a5d6619b5009b18f99dd111dc7c2dd32f0fae5:1:in Object.onPlaneViewRender
```

<a href="https://timer.scm.io/repos/537237ce1a00001a003c4a14/issues/2547/create?referer=github" target="_blank">Log Time</a>

**Comment by hotzenklotz** [comment]
@philippotto Do you mind taking a look at this. Something is going wrong in`calculateUnzoomedAnchorPoint`

**Comment by philippotto** [comment]
Should be fixed by #2545.
